### PR TITLE
Upgrade dependencies, misc. fixes

### PR DESCRIPTION
**Pull request by myabc**
This PR contains dependency updates (two commits taken from `develop`) + minimal fixes needed to get cxsd building. I'll leave refactorings/feature ideas to another PR.

### CODE REVIEW

File: src/schema/exporter/Exporter.ts [hunk 1]
```diff
@@ -41,12 +47,15 @@ export abstract class Exporter extends Transform<Exporter, boolean, State> {
 		if(!doc) return(null);
 
 		this.cacheDir = path.dirname(
-			this.state.cache.getCachePathSync(new Address(doc.namespace.name))
+			this.state.fsStrategy.getCachePathSync(doc.namespace.name)
 		);
+	
+		let outName = this.getOutName(doc.namespace.name);		
+		if (doc.namespace.isPrimitiveSpace || new Address(outName).isLocal) {
```
Comment (myabc) [inline src/schema/exporter/Exporter.ts:54]: @charto This is a big hack - Although I suspect there is a better way of doing this, this is the best I could come up with.

**Comment by jjrv** [comment]
Thanks! Could you remove the yarn.lock file? Then I can merge.

File: src/tsconfig.json [hunk 2]
```diff
@@ -8,7 +8,8 @@
 		"noImplicitThis": true,
 		"outDir": "../dist",
 		"removeComments": false,
-		"target": "es5"
+		"target": "es5",
+		"lib": ["es6"]
```
Comment (jjrv) [inline src/tsconfig.json]: This could be just: `[ "es5", "es2015.collection" ]`

**Comment by davidspiess** [comment]
Is there anything blocking this pull request?
The currently used cget version pulls in a vulnerable dependency.

image

**Comment by FranklinChen** [comment]
What's the status of this pull request?

**Comment by myabc** [comment]
I no longer have write access to the OUP repository so will need to open new PR

**Comment by myabc** [comment]
Replaced by PR #27

**Pull request closed without merging**